### DISCUSSION TITLE
Custo Keybindings

### DISCUSSION THREAD

**Discussion by shlozm** [regolith-linux/regolith-desktop#930] [General]
I added a custom keybinding for $mod+Shift+Return to launch my preferred Web Browser which is an AppImage of Thorium, added to .config/regolith3/Xresources, nothing happens when I try that key combination. I used the usual I3 syntax bindsym $mod+Shift+Return exec --no-startup-id  /home/username/.local/bin/Thorium.AppImage
It launches fine from the terminal and Rofi
Please help , thanks

**Comment by kuhlekt1v** [comment]
@shlozm were you ever able to figure this out? I’m facing a similar issue, but tried creating a new keybind in a [staged config](https://regolith-desktop.com/docs/howtos/stage-configs/).

**Comment by Will6834** [comment]
In case anyone else has this problem:
The default config files for Regolith 3.3 determine the default web browser by executing the command `xdg-settings get default-web-browser`, and bind it to $Mod+Shift+Enter. So, you could use `xdg-settings` to specify a different default web browser. But you can also use the Settings app. ($Mod+C)

The config file that contains this setting is `/usr/share/regolith/common/config.d/15_base_launchers` (in the default setup).
